core/src/components/toggle/toggle.md.scss

```
@@ -33,73 +36,28 @@
 }
 
 // Material Design Toggle Background Track: Unchecked
-// -----------------------------------------
+// ----------------------------------------------------------
 
 .toggle-icon {
-  @include border-radius(var(--border-radius));
-
-  display: block;
-  position: relative;
-
-  width: 100%;
-  height: 100%;
-
   transition: background-color $toggle-md-transition-duration;
-
-  background: var(--background);
-  pointer-events: none;
 }
 
 
 // Material Design Toggle Inner Knob: Unchecked
-// -----------------------------------------
+// ----------------------------------------------------------
 
 .toggle-inner {
-  @include position(($toggle-md-handle-height - $toggle-md-track-height) / -2, null, null, 0);
-  @include border-radius(var(--handle-border-radius));
-
-  position: absolute;
-
-  width: $toggle-md-handle-width;
-  height: $toggle-md-handle-height;
-
-  transition-duration: $toggle-md-transition-duration;
-  transition-property: transform, background-color;
-  transition-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
-
-  background: var(--handle-background);
-
-  box-shadow: $toggle-md-handle-box-shadow;
-  will-change: transform, background-color;
-  contain: strict;
-}
-
-
-// Material Design Toggle Background Track: Checked
-// -----------------------------------------
-
-:host(.toggle-checked) .toggle-icon {
-  background: var(--background-checked);
-}
-
-// Material Design Toggle Inner Knob: Checked
-// -----------------------------------------
-
-:host(.toggle-checked) .toggle-inner {
-  @include transform(translate3d($toggle-md-track-width - $toggle-md-handle-width, 0, 0));
-
-  background: var(--handle-background-checked);
+  will-change: background-color;
```

```suggestion
  will-change: background-color, transform;
```

Looks like the transform changes on toggle-inner too: https://github.com/ionic-team/ionic/pull/21050/files#diff-06267e2174a1a50220f0594452a3d09cR130